Using perkstore - screenshots missing (#136)
exoplatform/exo-documentation: docs/UsingPerkStore.rst

.. _UsingPerkStore:

####################

> Using the Perkstore

Reviewer: ```suggestion
Using the Perk Store
```

####################

> The perkstore is an online store available in the platform. 

Reviewer: ```suggestion
The Perk Store is a digital store available in the platform where users can purchase perks (or whatever goods your organization wants to offer to its users) with their earned funds. 
```

> You are able to buy the available products or sell goods or services using the cauris as a currency.

Reviewer: ```suggestion
It’s a full-featured marketplace where users can buy (and even sell) products with the tokens stored in their wallet as the digital currency.
```

> How to access to Perkstore

Reviewer: ```suggestion
How to access to Perk Store
```
```suggestion
How to access to Perkstore
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~

> You can access to perkstore from the left navigation. By clicking on the appropriate icon, you will display all the available products in the store.

Reviewer: ```suggestion
You can access to Perk Store from the main navigation. By clicking on the shopping cart icon, you will see all the products available for you in the store.
```

> You can see your wallet balance in the toolbar as well as a search bar and a filter button that allow you to look for a specific product.

Reviewer: ```suggestion
You can see your wallet balance in the toolbar as well as a search bar and a filter button to help you find a specific product.
```

|image0|

> You can filter the display of articles by:

Reviewer: ```suggestion
You can filter products by:
```

> - Disable products

Reviewer: ```suggestion
- Disabled products : some products may be temporarily disabled by their seller
```

> - Soldout products

Reviewer: ```suggestion
- Sold out products : products that are no longer in store
```

Author: The products that are no longer available in stock.
WDYT?

> - Created by me

Reviewer: ```suggestion
- Created by me : the products you offered for sale yourself
```

> You can see more details about a product by clicking on its title. You can buy the product from this view as well as the perkstore view.

Reviewer: ```suggestion
You can see more details about a product by clicking on its title. You can buy the product from this screen as well as the Perk Store screen.
```

|image1|

How to buy a product and display your orders
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> To buy a product you are interested in, you have to click on the appropriate button |image2| then specify the number of the items to purchase as well as your wallet password if it is not saved in the browser you are using.

Reviewer: ```suggestion
To purchase a product, click on the Buy button button |image2| then specify the number of the items to purchase. Your password may be required unless you checked the “Remember my password” option in your wallet’s preferences.
```

|image3|

> After buying the product, You recieve a notification when your order is totally or partially delivered.

Reviewer: ```suggestion
After buying the product, a notification will tell you when your order is totally or partially delivered.
```

|image4|

> In the perkstore view a new mention "1 pending" is available for you in the purchased product's card 

Reviewer: ```suggestion
In the Perk Store screen a new mention "1 pending" is available for you in the purchased product's card 
```

|image5|

> You can see your order details for this product by clicking in this button.

Reviewer: ```suggestion
You can see your order details for this product by clicking on this button.
```

|image6|

> To see all orders you can click on "My orders" button from the toolbar of perkstore.

Reviewer: ```suggestion
To see all orders you can click on "My orders" button from the toolbar of the Perk Store.
```

|image7|

|image8|

How to create a product
~~~~~~~~~~~~~~~~~~~~~~~

> If you are allowed to create products from perkstore, you will see the creation card as the first card displayed in the interface. 

Reviewer: ```suggestion
If you are allowed to create products , you will see the “Plus” card as the first card displayed in the interface. 
```

|image9|

> By clicking on it, the creation form is displayed.

Reviewer: ```suggestion
By clicking on it, the Product creation form is displayed.
```

|image10|

> You have to fill in the mandatory fields which are:

Reviewer: ```suggestion
Mandatory fields which are:
```

- The name of the product, 

> - The selling price: The product's price in Cauris

Reviewer: ```suggestion
- The sell price: The product's price in token currency value
```

> - The merchant wallet: The user's or space's wallet that will ecieve the funds when the product is purchased, and

Reviewer: ```suggestion
- The merchant wallet: The user's or space's wallet that will receive the funds when the product is purchased, and
```

> - The product editor: This field is automatically pre-loaded with the current user account and you have the possibilityto change this.

Reviewer: ```suggestion
- The product editor: This field is automatically pre-loaded with your user account but you can change it if your want to designate another beneficiary for the product sales revenues

.
```

Author: The product editor has the right to edit the product but doesn't receive funds, the funds are sent to the merchant wallet

You can also optionnaly upload images, add a description of the product and select authorized buyers (users or spaces members).

> You may also specify the stock limit and to restrict the number of purchased items by user and per periodicity.

Reviewer: ```suggestion
You may also specify a stock limit and to restrict the number of purchased items by user and per periodicity.
```

|image11|

|image12|

Once your product is sold, you will recieve a notification.

|image13|

> You can see the client's order by clicking on the notification or clicking on the following icon |image14| from the perkstore interface.

Reviewer: ```suggestion
You can inspect your clients orders by clicking on the notification or by clicking on the following icon |image14| from the Perk Store interface.
```
